File: apis/csharp/routine-api/exception/ClientCommunicationException.cs
using System;
namespace routine_api.exception
{
    public class ClientCommunicationException : Exception
    {
        public ClientCommunicationException(string message) : base(message)
        {
        }

        public ClientCommunicationException(Exception cause) : base(cause.Message, cause)
        {
        }

        public ClientCommunicationException(string message, Exception cause) : base(message, cause)
        {
        }
    }
}

File: communication-api/src/main/java/at/enfilo/def/communication/exception/ClientCommunicationException.java
package at.enfilo.def.communication.exception;

/**
 * Created by mase on 31.08.2016.
 */
public class ClientCommunicationException extends Exception {

    public ClientCommunicationException(String message) {
        super(message);
    }

    public ClientCommunicationException(String message, Throwable cause) {
        super(message, cause);
    }

    public ClientCommunicationException(Throwable cause) {
        super(cause);
    }

    public ClientCommunicationException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace) {
        super(message, cause, enableSuppression, writableStackTrace);
    }
}
